src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +22,49 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { form } = this.props;
+    const { response } = form.submission;
```

Comment:
I would make sure this page renders without console errors if you refresh it without submitting the form again.

Reply:
Thanks @jbalboni, changed.